Integrations/integration-DemistoRESTAPI.yml

```diff
@@ -119,6 +119,29 @@ script:
         }
     };
 
+    var deleteIncident = function(ids_to_delete) {
```

Comment:
the function deletes multiple incidents so the name should be plural: `deleteIncidents`

Reply:
Fixed